packages/studio-ui/src/components/EditStreamScopeModal.tsx

```diff
@@ -5,7 +5,8 @@ import StreamScopePicker, {
 } from "./StreamScopePicker";
 import { ResponseType, StreamScope } from "@yext/studio-plugin";
 import useStudioStore from "../store/useStudioStore";
-import { isEqual, sortBy } from "lodash";
+import isEqual from "lodash/isEqual";
+import sortBy from "lodash/sortBy";
```

Comment:
I think Oliver was mentioning in an [earlier PR](https://github.com/yext/studio/pull/374#discussion_r1329124742) that lodash per method packages are less efficient than just importing it wholesale according to [lodash](https://lodash.com/per-method-packages).

Reply:
My understanding of what lodash is saying there is that they don't recommend installing the individual per-method packages (e.g. `lodash.isEqual`) and you should import from the overall `lodash` package like we're doing here (their example is `const throttle = require('lodash/throttle')`)

Reply:
ohh I see I think I misunderstood what they were saying, that makes sense